### DISCUSSION TITLE
Copilot is encountering temporary issues (Enterprise)

### DISCUSSION THREAD

**Discussion by tzhgeco1cg** [community/community#49488] [Enterprise]
### Select Topic Area

Bug

### Body

After successful login, as soon as I start coding, the icon in IntelliJ for CoPilot is strikethrough and shows "Copilot is encountering temporary issues"

I checked already:
Proxy Settings
Reinstalled
Disabeld and enabled CoPilot
Logout, Login
Checked Company Settings
Checked Timezone on Mac

But still I get always this message when I'm coding

**Comment by runesatsdev** [comment]
Hi
If none of the above solutions work, contact GitHub support to report the issue and get further assistance. You can visit the GitHub website for support options, including email, chat, and community forums.
Best of luck.

**Comment by 0xNevo** [comment]
Based on the information provided, it seems like there might be server-side issues with the Copilot Enterprise service. As you have already tried several troubleshooting steps, I suggest reaching out to the support team of Copilot Enterprise service, describing the issue you are encountering and the steps you've taken so far.

Here are a few points to consider while reaching out to them:

Share the exact error message and any accompanying error codes.
Mention the IDE (IntelliJ) version, Copilot version, Operating System, and other relevant details that can help them identify the problem.
Provide your account details and company credentials if necessary.
Be specific in explaining the steps you took to troubleshoot the issue and the outcomes of those attempts.
Ask for an Estimated Time of Resolution (ETR) and contact information to talk or track progress.
With this information, the support team should be able to provide you with better assistance regarding the temporary issues with the Copilot Enterprise service.

**Reply by tzhgeco1cg** [reply]
I contacted the support team
